src/main/java/io/vertx/redis/client/Response.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
+ * <p>
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Apache License v2.0 which accompanies this distribution.
+ * <p>
+ * The Eclipse Public License is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * <p>
+ * The Apache License v2.0 is available at
+ * http://www.opensource.org/licenses/apache2.0.php
+ * <p>
+ * You may elect to redistribute this code under either of these licenses.
+ */
+package io.vertx.redis.client;
+
+import io.vertx.codegen.annotations.GenIgnore;
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.buffer.Buffer;
+
+import java.nio.charset.Charset;
+import java.util.Iterator;
+import java.util.Set;
+
+@VertxGen
+public interface Response extends Iterable<Response> {
```

Comment:
missing javadoc, also the javadoc should explain why a response is an iterable of response (recursion ?)